src/bin/bobp.rs

```diff
@@ -100,7 +100,7 @@ fn bench_worker(net_conf: NetConfig, task_conf: TaskConfig, stat: Arc<Stat>) {
                 .unwrap();
     let mut client = BobApi::new(p_conn.clone());
     for i in task_conf.low_idx..task_conf.high_idx {
-        let pur_req =  client.put(Request::new(PutRequest { key: None, data: None, options: None}))
+        let pur_req =  client.put(Request::new(PutRequest {key: Some(BlobKey {key:i}), data: Some(Blob {data: vec![0; task_conf.payload_size as usize]}), options: None}))
```

Comment:
I suggest to use `cargo fmt`, maybe in separate commit

Reply:
Sure